src/util/subprocess.hpp

```diff
@@ -1336,11 +1211,6 @@ inline void Popen::execute_process() noexcept(false)
 {
 #ifdef __USING_WINDOWS__
   void* environment_string_table_ptr = nullptr;
```

Comment:
In 0f847336428153042dbf226caf52ecd43188f22e "remove unneeded environment option from cpp-subprocess"

This variable seems to also be unused now as it is never set to anything. It's only passed in as an argument to `CreateProcessW` which could take the nullptr directly.